### PR TITLE
Feature/#230 experience and studies sorted by date

### PR DESCRIPTION
**Pull request by migalko94**

### CODE REVIEW

File: packages/manfred-common/src/doc-parts/experience-section/experience-section.helpers.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { ExperienceVm } from './experience-section.vm';
+import { dateExtractor, sortByDate } from '../../helpers';
```
Comment (juanpms2) [inline packages/manfred-common/src/doc-parts/experience-section/experience-section.helpers.ts:2]: Use '@/helpers' on import and move it at first line

File: packages/manfred-common/src/doc-parts/experience-section/experience-section.helpers.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+export const sortRolesByDate = (experience: ExperienceVm): ExperienceVm => ({
+  ...experience,
+  roles: sortByDate(experience.roles, 'startDate'),
+});
+
+export const mapSortedRolesIntoExperience = (experience: ExperienceVm[]): ExperienceVm[] =>
+  experience.map(experienceItem => sortRolesByDate(experienceItem));
+
+export const sortExperienceByDate = (experience: ExperienceVm[]): ExperienceVm[] =>
```
Comment (juanpms2) [inline packages/manfred-common/src/doc-parts/experience-section/experience-section.helpers.ts:12]: What's the difference between sortExperienceByDate and sortByDate functions?

File: packages/manfred-common/src/doc-parts/experience-section/experience-section.mapper.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { ManfredAwesomicCV } from '../../model';
```
Comment (juanpms2) [inline packages/manfred-common/src/doc-parts/experience-section/experience-section.mapper.ts:1]: Use '@/model'

File: packages/manfred-common/src/doc-parts/studies-section/studies-section.helpers.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { sortByDate } from '../../helpers';
```
Comment (juanpms2) [inline packages/manfred-common/src/doc-parts/studies-section/studies-section.helpers.ts:1]: Use '@/helpers"

File: packages/manfred-common/src/doc-parts/experience-section/experience-section.helpers.ts (hunk 1 above)
Comment (migalko94) [reply packages/manfred-common/src/doc-parts/experience-section/experience-section.helpers.ts:2]: With import in this way I get test failures

File: packages/manfred-common/src/doc-parts/studies-section/studies-section.helpers.ts (hunk 4 above)
Comment (migalko94) [reply packages/manfred-common/src/doc-parts/studies-section/studies-section.helpers.ts:1]: Same as with experience. With import in this way I get test failures

**Comment by migalko94** [comment]
Added missing typing on 'b' in sortExperienceByDate function

**Pull request closed without merging**